docs/platform/howto/add-storage-space.rst

```diff
@@ -1,7 +1,7 @@
 Add or remove storage 
 =======================
 
-With :doc:`dynamic disk sizing <../concepts/dynamic-disk-sizing>`, you can add or remove disk storage both when you create a service and later for a running service. 
+With :doc:`dynamic disk sizing <../concepts/dynamic-disk-sizing>`, you can add or remove disk storage (by factor of 10 GiB only) both when you create a service and later for a running service. But, it is not possible to add or remove storage when the service nodes are rebalancing.(i.e, during maintenance update, service upgrade, etc.)
```

Comment:
1. Please remove "only" - no need to imply this is some limitation
2. Please include the last sentence as a note:

```
.. note::
   You cannot add or remove storage when service nodes are in the rebalancing state, for example, during a maintenance update or a service upgrade.
```